plugins/module_utils/util.py

```diff
@@ -376,7 +376,7 @@ def normalize_healthcheck(healthcheck, normalize_test=False):
                         'Cannot parse number of retries for healthcheck. '
                         'Expected an integer, got "{0}".'.format(value)
                     )
-            if key == 'test' and normalize_test:
+            if key == 'test' and value and normalize_test:
```

Comment:
This one also needs to be adjusted.

Reply:
I don't have this one, for me that code still never reach due to previous continue if value missing (keep the current behavior) and the only case I see is when test_cli_compatible is true and forward an empty test value.
Did I miss something ?

Reply:
No, everything's fine, I was too overcautious and didn't look closely enough :) The old code had `if not value: continue` further above, so in the old code path this `if` is never reached if `value` isn't truish.

Reply:
No problem always better to be overcautious than not enough ;)